src/js/constants.js

```diff
@@ -1,4 +1,9 @@
 export const STUY_SPEC_API = 'https://api.stuyspec.xyz';
 export const STUY_SPEC_API_HEADER = {
   'X-Key-Inflection': 'camel',
-};
\ No newline at end of file
+};
+
+export const MEDIA_CREATOR_SLUG = {
```

Comment:
It's what @ccai1 and @JasonLin43212 renamed it to. If you don't like it, can you propose a name? Because Cathy, Jason, and I have already taken whips at this.